glide.yaml

```diff
@@ -17,7 +17,7 @@ import:
 - package: github.com/kelseyhightower/envconfig
   version: ^1.3.0
 - package: golang.org/x/net
-  version: 3da985ce5951d99de868be4385f21ea6c2b22f24
+  version: 66aacef3dd8a676686c7ae3716979581e8b03c47
```

Comment:
are the envconfig and procfs pins required?

Reply:
never mind, looks like you're doing the same update in multiple repos, so I'll assume they're required.

Reply:
Yeah, the procfs one is due to a loose pin in a dependency causing problems.

The x/net one is required to get around a compile time issue around mismatched versions, as a result of revving etcd (which is required for revving libcalico)